src/atoms/cc-button.js

```diff
@@ -210,9 +211,14 @@ export class CcButton extends LitElement {
         ${delay != null ? html`
           <progress class="delay ${classMap({ active: this._cancelMode })}" style="--delay: ${delay}s"></progress>
         ` : ''}
-        ${waiting ? html`
+        ${waiting && !modes.circle ? html`
           <progress class="waiting"></progress>
         ` : ''}
+        ${waiting && modes.circle ? html`
+          <svg class="circle-loader" viewBox="25 25 50 50" stroke-width="4">
```

Comment:
Let's consider this `<svg>` is decorative because we will rely on something else (dunno what yet, see #486) to convey the "loading" status for screen reader users. 

This means that you should add an `aria-hidden="true"` attribute to the `<svg>` element:
`<svg ... aria-hidden="true">`

(I'll create a new "rule" in our a11y document to sum up how we handle `<svg>` elements, sorry for not doing it before :wink:)